libvast/src/system/explore_command.cpp

```diff
@@ -56,7 +56,7 @@ caf::message explore_command(const invocation& inv, caf::actor_system& sys) {
                   defaults::explore::max_events_query);
   // Get a local actor to interact with `sys`.
   caf::scoped_actor self{sys};
-  auto s = make_sink(sys, inv.options, output_format);
+  auto s = make_sink(sys, output_format, inv.options);
```

Comment:
Was there a specific reason for switching the argument order of `make_sink`? This is not a dealbreaker, just curious.

Reply:
I changed it so the order of arguments is the same as for `spawn_sink` and `factory<writer>::make`.